### PR TITLE
Add Switch component

### PR DESCRIPTION
**Pull request by maemreyo**
I would like to add a Switch component.
And export Skeleton and Switch as default.
Please take a look. 
Thanks

### CODE REVIEW

File: src/packages/Switch/styles.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      color: ${theme.colors.neutral.darkest};
+    `
+  }
+}
+
+export const Label = styled.label`
+  ${() => css`
+    display: flex;
+    align-items: center;
+    gap: 10px;
```
Comment (vczb) [inline src/packages/Switch/styles.ts:46]: use `rem` instead pixels

10px = 1rem

File: src/packages/Switch/styles.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+export const Input = styled.input<Pick<SwitchProps, 'colors' | 'disabled'>>`
+  ${({ colors, disabled }) => css`
+    opacity: 0;
+    position: absolute;
+
+    &:checked + ${Switch} {
+      ${figureModifiers['bgChecked'][disabled ? 'disabled' : 'normal'](colors)}
+
+      &:before {
+        transform: translate(30px, -50%);
```
Comment (vczb) [inline src/packages/Switch/styles.ts:59]: 3rem

File: src/packages/Switch/styles.ts [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+export const Span = styled.span<Pick<SwitchProps, 'colors' | 'disabled'>>`
+  ${({ disabled }) => css`
+    ${figureModifiers['label'][disabled ? 'disabled' : 'normal']()};
+  `}
+`
+
+export const Switch = styled.span<Pick<SwitchProps, 'colors' | 'disabled'>>`
+  ${({ colors, disabled }) => css`
+    position: relative;
+    width: 60px;
```
Comment (vczb) [inline src/packages/Switch/styles.ts:74]: apply the rem for all pixels

File: src/packages/Switch/styles.ts [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    height: 28px;
+    ${figureModifiers['bgUnchecked'][disabled ? 'disabled' : 'normal'](colors)}
+
+    border-radius: 32px;
+    padding: 4px;
```
Comment (vczb) [inline src/packages/Switch/styles.ts:79]: you can use `theme` here

`theme.spacings.mini`

File: src/packages/Switch/styles.ts [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+`
+
+export const Switch = styled.span<Pick<SwitchProps, 'colors' | 'disabled'>>`
+  ${({ colors, disabled }) => css`
+    position: relative;
+    width: 60px;
+    height: 28px;
+    ${figureModifiers['bgUnchecked'][disabled ? 'disabled' : 'normal'](colors)}
+
+    border-radius: 32px;
```
Comment (vczb) [inline src/packages/Switch/styles.ts:78]: `theme.spacings.medium`

**Review by vczb — COMMENTED** [review]
Very nice, lets use REM instead PIXELS

Also try to use the theme to styling 

Any question let me know

---

Important: Do your fixes and squash or amend the changes to avoid more then 1 commit per PR

File: src/packages/Switch/styles.ts (hunk 5 above)
Comment (vczb) [reply src/packages/Switch/styles.ts:78]: research other places where the theme can be applied too

**Comment by maemreyo** [comment]
I already create just one commit per PR, please take a look @vczb. Thanks!

**Review by vczb — APPROVED** [review]
LGTM

**Pull request merged**